frontend/package.json

```diff
@@ -26,7 +26,6 @@
    "dependencies": {
       "@chakra-ui/icons": "^1.0.15",
       "@chakra-ui/react": "1.8.8",
-      "@chakra-ui/system": "1.12.1",
```

Comment:
This looks unrelated to the rest of these changes. Was it intentional?

Reply:
It is intentional 👍 It's an unnecessary dependency, and I've removed it while fixing this behaviour